_source/logzio_collections/_log-sources/fluentd-windows.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Fluentd
+logo:
+  logofile: fluentd.svg
+  orientation: vertical
+short-description: Fluentd is a DaemonSet that collects and sends logs to Logz.io.
+data-source: Fluentd - Windows logs
+data-for-product-source: Logs
+shipping-tags:
+  - log-shipper
+contributors:
+  - nshishkin
+shipping-tags:
+  - agents
+order: 180
+---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Overview](#overview)
+* [Setup instructions](#setup-instructions)
+* [Multiline logs](#multiline)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="overview">
+
+Fluentd is a data collector, which unifies the data collection and consumption. This integration allows you to use Fluentd to send logs from your Windows system to your Logz.io account. 
+
+<!-- info-box-start:info -->
+Fluentd will fetch all existing logs, as it is not able to ignore older logs.
+{:.info-box.note}
+<!-- info-box-end -->
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="setup-instructions">
+
+
+#### Configure Fluentd
+
+**Before you begin, you'll need**:
+Ruby and ruby-dev 2.1 or higher
+
+<div class="tasklist">
+
+	
+##### Install Fluentd td-agent
+
+Navigate to the [downloads](https://td-agent-package-browser.herokuapp.com/4/windows) page of td-agent and download the latest version of the installer. After that, run the installer and follow the wizard instructions.
+
+##### Install the Logz.io plugin
+
+```shell
+gem install fluentd fluent-plugin-logzio
+```
+
+##### Set up td-agent.conf
+
+Open C:/opt/td-agent/etc/td-agent/td-agent.conf and replace its contents with the following configuration:
+
+```xml
+<source>
+  @type windows_eventlog2
+  @id windows_eventlog2
+  channels application,system,security
+  read_existing_events false
+  tag winevt.raw
+  rate_limit 200
+  <storage>
+    @type local
+    persistent true
+    path C:\opt\td-agent\winlog.json
+  </storage>
+</source>
+
+<match **>
+  @type logzio_buffered
+  endpoint_url <<LISTENER-HOST>>:8071?token=<<LOG-SHIPPING-TOKEN>>&type=<<LOG-TYPE>>
```

Comment:
Since its an endpoint url we need the full url with the https prefix
Should be:
https://<<LISTENER-HOST>>:8071?token=<<LOG-SHIPPING-TOKEN>>&type=<<LOG-TYPE>>